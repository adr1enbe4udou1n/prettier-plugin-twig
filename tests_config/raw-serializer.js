const RAW = Symbol.for("raw");

const print = val => val[RAW];
const test = val =>
    val &&
    Object.prototype.hasOwnProperty.call(val, RAW) &&
    typeof val[RAW] === "string";
export default {
    print,
    test
};
