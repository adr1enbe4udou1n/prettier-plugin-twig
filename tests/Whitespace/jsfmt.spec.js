import { run_spec } from "tests_config/run_spec";
import { describe, expect, it } from "vitest";

/** @type {import('tests_config/run_spec').PrettierOptions} */
const formatOptions = {};

describe("Whitespace", () => {
    it("should handle whitespace", async () => {
        const { actual, snapshotFile } = await run_spec(import.meta.url, {
            source: "element.twig"
        });
        await expect(actual).toMatchFileSnapshot(snapshotFile);
    });
});
