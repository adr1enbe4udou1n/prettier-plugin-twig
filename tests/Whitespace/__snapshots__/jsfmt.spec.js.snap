// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`element.melody.twig - melody-verify > element.melody.twig 1`] = `
<div>
    <img src="person.jpg">
</div>

<p>
    This is some text This is some text This is some text This is some text This is some text This is some text This is some text This is some text This is some text 
</p>

<div class="striped-rows">
    <!-- Pregunta 1 -->
    {% include 'pages/formularios/sub/pregunta-simple.html' %}
    {% set index = index + 1 %}
</div>
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
<div>
    <img src="person.jpg" />
</div>

<p>
    This is some text This is some text This is some text This is some text This
    is some text This is some text This is some text This is some text This is
    some text
</p>

<div class="striped-rows">
    <!-- Pregunta 1 -->
    {% include 'pages/formularios/sub/pregunta-simple.html' %}
    {% set index = index + 1 %}
</div>

`;
